prov/efa/src/rxr/rxr_cq.c

```diff
@@ -391,8 +391,8 @@ int rxr_cq_handle_cq_error(struct rxr_ep *ep, ssize_t err)
 		rxr_ep_dec_tx_pending(ep, peer, 1);
 	if (RXR_GET_X_ENTRY_TYPE(pkt_entry) == RXR_TX_ENTRY) {
 		tx_entry = (struct rxr_tx_entry *)pkt_entry->x_entry;
-		if (err_entry.err != -FI_EAGAIN ||
-		    rxr_ep_domain(ep)->resource_mgmt == FI_RM_ENABLED) {
```

Comment:
Can you remove `resource_mgmt` from `rxr_domain`? I believe this is the only place it is used.